sql/src/main/java/io/crate/analyze/RestoreSnapshotAnalyzedStatement.java

```diff
@@ -23,29 +23,32 @@
 package io.crate.analyze;
 
 import com.google.common.collect.ImmutableList;
+import io.crate.metadata.PartitionName;
+import io.crate.metadata.TableIdent;
+import org.elasticsearch.common.Nullable;
 import org.elasticsearch.common.settings.Settings;
 
 import java.util.List;
 
 public class RestoreSnapshotAnalyzedStatement implements DDLStatement {
 
-    private static final List<String> ALL_INDICES = ImmutableList.of();
+    private static final List<RestoreTableInfo> ALL_INDICES = ImmutableList.of();
```

Comment:
`ALL_INDICES` doesn't fit anymore as name - rename to something like `EVERYTHING`, `ALL`, `ALL_TABLES`